config/canonical.json

```diff
@@ -4464,14 +4486,21 @@
     "tags": {
       "amenity": "bank",
       "brand": "Уралсиб",
-      "name": "Уралсиб"
+      "brand:en": "Uralsib bank",
+      "brand:wikidata": "Q4476693",
+      "brand:wikipedia": "ru:Уралсиб (банк)",
+      "name": "Уралсиб",
+      "name:en": "Uralsib bank"
     }
   },
   "amenity/bank|Хаан банк": {
     "count": 52,
+    "countryCodes": ["mn"],
     "tags": {
       "amenity": "bank",
       "brand": "Хаан банк",
+      "brand:wikidata": "Q25632240",
```

Comment:
This wikidata entry is really bad, but I suppose there's nothing we can do about that

Reply:
I just added "instance of : bank" and "country : Mongolia", but I couldn’t infer much more from the wikipedia article :/

Reply:
Nice!